### PR TITLE
Migrate Unit Tests to phpunit 8

### PR DESCRIPTION
**Pull request by lbajsarowicz**
### Description (*)
TLDR: magento/magento2#27500

### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#<issue_number>: Issue title
2. ...

### Manual testing scenarios (*)

1. ...
2. ...

### CODE REVIEW

File: AdobeIms/Test/Unit/Controller/Adminhtml/User/ProfileTest.php [hunk 1]
```diff
@@ -8,18 +8,18 @@
...
 use Magento\Framework\Exception\NotFoundException;
 use PHPUnit\Framework\MockObject\MockObject;
-use Psr\Log\LoggerInterface;
-use Magento\AdobeIms\Controller\Adminhtml\User\Profile;
-use Magento\Framework\Controller\Result\Json;
 use PHPUnit\Framework\TestCase;
+use Psr\Log\LoggerInterface;
 
 /**
  * Ensure that User Profile data can be returned.
```
Comment (coderimus) [inline AdobeIms/Test/Unit/Controller/Adminhtml/User/ProfileTest.php:25]: Maybe we should have the `SuppressWarnings(PHPMD.CouplingBetweenObjects)` also here?

File: AdobeStockAsset/Test/Unit/Model/SaveAssetTest.php [hunk 2]
```diff
@@ -8,17 +8,17 @@
...
-use Magento\AdobeStockAssetApi\Api\CategoryRepositoryInterface;
-use PHPUnit\Framework\MockObject\MockObject;
 use Magento\Framework\Reflection\DataObjectProcessor;
-use PHPUnit\Framework\TestCase;
 use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
 
 /**
  * Test for save asset service.
```
Comment (coderimus) [inline AdobeStockAsset/Test/Unit/Model/SaveAssetTest.php:24]: Should we add the `SuppressWarnings(PHPMD.CouplingBetweenObjects)` to this test class also?

**Review by coderimus — CHANGES_REQUESTED** [review]
Hello @lbajsarowicz ! Thank you for your contribution. Really good to have project tests adopted to the PHPUnit 8. I have two questions regarding the `SuppressWarnings(PHPMD.CouplingBetweenObjects)`. Please, check them.

File: AdobeIms/Test/Unit/Controller/Adminhtml/User/ProfileTest.php (hunk 1 above)
Comment (lbajsarowicz) [reply AdobeIms/Test/Unit/Controller/Adminhtml/User/ProfileTest.php:25]: Static checks are not reporting any issues. Why would you like to do so?

File: AdobeStockAsset/Test/Unit/Model/SaveAssetTest.php (hunk 2 above)
Comment (lbajsarowicz) [reply AdobeStockAsset/Test/Unit/Model/SaveAssetTest.php:24]: Static checks are not reporting any issues. Why would you like to do so?

Comment (coderimus) [reply AdobeStockAsset/Test/Unit/Model/SaveAssetTest.php:24]: Yep, you are right. No, reported issue. I checked out to the magento2/phpunit8 branch to test this PR. Everything fine. Thank you for your work @lbajsarowicz

**Review by coderimus — DISMISSED** [review]

**Review by slavvka — CHANGES_REQUESTED** [review]
@lbajsarowicz please fix the static tests

**Comment by lbajsarowicz** [comment]
@coderimus / @slavvka Fixed.

Why ASI has different set of Static tests than core?

**Comment by slavvka** [comment]
> Why ASI has different set of Static tests than core?

What do you mean?

**Review by slavvka — APPROVED** [review]

**Pull request merged**